libs/shared/ui/src/lib/components/inputs/input-text/input-text.tsx

```diff
@@ -11,10 +11,11 @@ export interface InputTextProps {
   error?: string
   disabled?: boolean
   dataTestId?: string
+  rightFloatingComponent?: React.ReactNode
 }
 
 export function InputText(props: InputTextProps) {
-  const { name, label, value, onChange, type = 'text', error, dataTestId, className = '', disabled } = props
+  const { name, label, value, onChange, type = 'text', error, className = '', disabled, rightFloatingComponent, dataTestId } = props
```

Comment:
Rename rightFloatingComponent to rightElement ? it seems long and floating doesn't seem to be the right word